python/hail/tests/test_expr.py

```diff
@@ -89,6 +89,9 @@ def test_bind_multiple(self):
         self.assertEqual(hl.bind(lambda x, y: x * y, 2, 3).value, 6)
         self.assertEqual(hl.bind(lambda y: y * 2, 3).value, 6)
 
+    def test_bind_placement(self):
```

Comment:
without the parens, does the parser get confused because it sees something like `5 / let x = 5 in x`?

Reply:
yep